### ISSUE TITLE
Updates on `PECA` and related repositories

### ISSUE THREAD

**Issue by t-silvers** [SUwonglab/PECA#17]
Hi there,

I've been exploring a bunch of repositories that either contain the `PECA` software or have `PECA` as a major dependency. I've found these repos across a few accounts:

1. [PECA by SUwonglab](https://github.com/SUwonglab/PECA)
2. [PECA by durenzn](https://github.com/durenzn/PECA)
3. [PECA-1 by durenzn](https://github.com/durenzn/PECA-1)
4. [TimeReg by SUwonglab](https://github.com/SUwonglab/TimeReg)
5. [vPECA by AMSSwanglab](https://github.com/AMSSwanglab/vPECA)

I've previously submitted issues on the [SUwonglab/PECA issue tracker](https://github.com/SUwonglab/PECA/issues) and made an attempt to get in touch with @durenzn directly, both via "@" and by email. Unfortunately, those issues are still awaiting responses.

To get a better picture of the current state of `PECA` and cast a wide net, I'm posting this identical issue across all these repositories. Is `PECA` archived, or is it still under development or maintenance? If it's still being developed or maintained, where's that mainly happening?

Any information or guidance would be super helpful. Thanks!

**Comment by durenzn** [comment]
Yes, PECA is still active and undergoing maintenance. I apologize for not seeing your comments earlier as I have been on summer vacation. Please review all my previous responses to your comments. If you have any further questions, kindly send an email to me(zduren@clemson.edu). Thank you.

**Comment by t-silvers** [comment]
No worries, hope you enjoyed your vacation 🍹!

I'll take a look at your replies to the other issues and follow up if needed. Before closing this issue and all its copies, would you clarify which `PECA` repository represents the "active" code base? Is it this one?
